### PR TITLE
Editing pass on intro and frontmatter

### PR DESCRIPTION
**Pull request by jeromekelleher**
Notes:

1. I changed the title, as the word "pain" didn't seem like a good thing to have in there. I'll open an issue for discussion.
2. I got rid of the ``\texttt{Demes}`` typography, as this won't survive final typesetting anyway. I think it's fine if we just call the specification Demes, and we can disambiguate where necessary by referring to ``\texttt{demes-python}`` or the ``\texttt{demes}`` Python module.

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -194,18 +266,41 @@ \subsection*{Unambiguous low-level representation}
...
+
+    \item Drawing parameters from distributions. Why we are static.
+    % https://github.com/popsim-consortium/demes-paper/issues/3
+
+    \item Why we've avoided turning this into a model of everything, allowing
+    for the specification of genome properties like recombination rates,
+    etc.
+
+    \item
+    % JK: I thought there was another simulation frontend somewhere..
```
Comment (grahamgower) [inline paper.tex:294]: stdpopsim?

File: paper.tex [hunk 2]
```diff
@@ -54,49 +76,99 @@ \section*{Introduction}
...
+using either a command line
+% JK: is there a simulator that just uses a CLI that's not using ms format?
+interface~\citep[e.g.][]{hudson2002generating},
+a custom input file format~\citep[e.g.][]{excoffier2011fastsimcoal,guillaume2006nemo},
+or an Application Programming Interface (API)~\citep[e.g.][]{
+thornton2014cpp,kelleher2016efficient,haller2019slim}.
+Command line interfaces are a concise way of expressing
+demographic models, and the syntax defined by \ms~\citep{hudson2002generating}
+is used by a number of different
+simulators~\citep[e.g.][]{ewing2010msms,staab2015scrm},
```
Comment (grahamgower) [inline paper.tex]: MaCS. Paper: https://genome.cshlp.org/content/19/1/136. Code: https://github.com/gchen98/macs

File: paper.tex [hunk 3]
```diff
@@ -54,49 +76,99 @@ \section*{Introduction}
...
+large number of different methods have been developed
+over the past three decades~\citep{carvajal2008simulation,liu2008survey,
+arenas2012simulation,yuan2012overview,hoban2012computer}.
+Simulations are based on highly idealised population models
+and one of the key uses of inferred demographic histories
+is to make simulations more realistic.
+Simulation methods take three broad approches to specifying
+the demographic model to simulate,
+using either a command line
+% JK: is there a simulator that just uses a CLI that's not using ms format?
```
Comment (grahamgower) [inline paper.tex]: Maybe [discoal](https://github.com/kr-colab/discoal)? `-ed` and `-ea` for population join and split don't look like ms commands to me.

File: paper.tex [hunk 4]
```diff
@@ -54,49 +76,99 @@ \section*{Introduction}
...
+interface~\citep[e.g.][]{hudson2002generating},
+a custom input file format~\citep[e.g.][]{excoffier2011fastsimcoal,guillaume2006nemo},
```
Comment (grahamgower) [inline paper.tex]: Cosi2. Paper: https://academic.oup.com/bioinformatics/article/30/23/3427/207410. Code: https://github.com/broadinstitute/cosi2

**Review by grahamgower — APPROVED** [review]
Nice!

File: paper.tex (hunk 1 above)
Comment (jeromekelleher) [reply paper.tex:294]: I suppose it is, but that's not what I'm getting at. I thought there was some other effort at some point to make a single frontend for a bunch of simulators to simplify running simulations. The point I want to make is that this is the wrong way to go - we shouldn't be writing code to resolve the differences between the simulators and to try and abstract their models of demography. We should all agree on a model of demography, and get the individual simulators to implement it.

**Comment by jeromekelleher** [comment]
Thanks for the review @grahamgower, I've updated with your comments.

**Review by apragsdale — APPROVED** [review]
LGTM

**Pull request merged**